packages/manager/cypress/support/intercepts/network-transfer.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * @file Cypress intercepts and mocks for Cloud Manager Object Storage operations.
+ */
+
+import { sequentialStub } from 'support/stubs/sequential-stub';
+import { apiMatcher } from 'support/util/intercepts';
+import { paginateResponse } from 'support/util/paginate';
+
+import type { PriceType } from '@linode/api-v4';
+
+/**
+ * Intercepts GET requests to fetch network-transfer prices and mocks response.
+ *
+ * Only returns data for the first request intercepted.
+ *
+ * @param priceTypes - Object storage buckets with which to mock response.
+ *
+ * @returns Cypress chainable.
+ */
+export const mockGetPrices = (
+  priceTypes: PriceType[]
+): Cypress.Chainable<null> => {
+  /*
+   * Only the first mocked response will contain data. Subsequent responses
+   * will contain an empty array.
+   *
+   * This is necessary because the Object Storage Buckets landing page makes
+   * an indeterminate number of requests to `/object-storage/buckets/<region>`,
+   * where `<region>` may be any region where Object Storage is supported.
+   */
+  return cy.intercept(
+    'GET',
+    apiMatcher('network-transfer/prices?page_size=500'),
+    sequentialStub([paginateResponse(priceTypes), paginateResponse([])])
```

Comment:
```suggestion
    apiMatcher('network-transfer/prices*'),
    paginateResponse(priceTypes),
```

- We can use a wildcard in the API endpoint so it matches any requests to `network-transfer/prices` regardless of the query params (but without matching any requests to `/network-transfer/prices/any-other-endpoints`)
- We don't need `sequentialStub` for this. That's only useful in situations where Cloud is making multiple requests to fetch some data and we don't know in advance how many requests will get made (e.g. fetching OBJ buckets since we fire a request for each cluster -- in fact that might be the only example).

I think both of these points apply to your `mockGetObjectStorageTypes` util as well, but otherwise these look great!